napari/_tests/test_config.py

```diff
@@ -0,0 +1,490 @@
+import os
+import stat
+import sys
+import shutil
+import tempfile
+from collections import OrderedDict
+from contextlib import contextmanager
+
+import pytest
+
+import napari.config  # noqa
```

Comment:
what's the dealio here?

Reply:
also removed... I'm not sure why my linter previously thought it wasn't used... cause it is used :)